site/src/sphinx/server-access-log.rst

```diff
@@ -120,6 +120,12 @@ Tokens for the log format are listed in the following table.
 +---------------------------+-------------------+----------------------------------------------------+
 | Tokens                    | Condition support | Description                                        |
 +===========================+===================+====================================================+
+| ``%A``                    | No                | the local IP address                               |
++---------------------------+-------------------+----------------------------------------------------+
+| ``%A``                    | No                | the remote IP address. The underlying client IP of |
```

Comment:
- Do the local IP address and the remote IP address have the same token, `%A`?
- How about:
  - The underlying client IP ... -> It represents the IP address of the proxy server if there's a proxy server between the client and the server. Use the `%{c}a` format string to get the real client IP address.